analytics/policy/policies/registration_number_policies.py

```diff
@@ -0,0 +1,52 @@
+import re
+import json
+from policy.services.business_licence_client import BusinessLicenceClient
+
+class RegistrationNumberPolicies:
+    def __init__(self, registration_number):
+        self.registration_number = registration_number
+
+    def valid_registration_number_policy(self):
+        valid_registration_pattern = re.compile(r'^[0-9]{2}-[0-9]{6}$')
+        START_YEAR = 13
+        END_YEAR = 24
+
+        if self.registration_number is None:
+            return False
+
+        if valid_registration_pattern.match(self.registration_number):
+            year = self.registration_number[0:2]
+            if int(year) > END_YEAR or int(year) < START_YEAR:
+                return False
+            else:
+                return True
+        else:
+            return False
+
+    def unique_registration_number_policy(self):
+        business_licences = BusinessLicenceClient()
+        licence_status = business_licences.get_licence_status(
+            self.registration_number
+        )
+        total_count = len(licence_status)
+        if total_count == 1:
+            if licence_status[0] == "Issued":
```

Comment:
sorry i forgot, the licence status result should not be checked here.
it shouldn't matter whether it's "Issued", "Cancelled",... as only uniqueness is checked